code/modules/events/spontaneous_appendicitis.dm

```diff
@@ -1,6 +1,8 @@
 /datum/event/spontaneous_appendicitis/start()
 	for(var/mob/living/carbon/human/H in shuffle(GLOB.living_mob_list_))
 		if(H.client && H.stat != DEAD)
+			if(prob(50))
+				continue
```

Comment:
bruh

Reply:
> ![image](https://user-images.githubusercontent.com/41485301/220433728-cf495fee-07dd-4713-9b3f-d4c82d119944.png)

The corrupted characters weren't changed from the original, didn't know that they were dysfunctional. Fixing.

Reply:
partially improved